RezoningScraper/API.cs

```diff
@@ -12,10 +13,14 @@ public static class API
 	/// Get all projects from the ShapeYourCity API.
 	/// </summary>
 	/// <returns>An async enumerable of projects (because the API is paginated)</returns>
-	public static async IAsyncEnumerable<Project> GetAllProjects(string jwt)
+	public static async IAsyncEnumerable<Project> GetAllProjects(string jwt, bool useCache = false)
 	{
+
+			IAsyncPolicy<Projects> cachePolicy = useCache
+				? Policy.CacheAsync<Projects>(new CacheManager<Projects>(), TimeSpan.FromHours(1))
+				: Policy.NoOpAsync<Projects>();
+			var client = new HttpClient();
 		string startUrl = $"https://shapeyourcity.ca/api/v2/projects?per_page={ResultsPerPage}";
```

Comment:
Mind fixing up the formatting here? New code is indented more than the existing code

Reply:
Yep, I've been writing a lot of python professionally and have been spoiled by black auto-formatting everything on save.